_i18n/en/resources/user-guides/remote_node_gui.md

```diff
@@ -26,7 +26,7 @@ The main menu (`Welcome to Monero` screen) will open. At the bottom left, click
 
 ## Finding a public remote node
 
-First, you will need to find a public remote node to connect to. The website [moneroworld.com](https://moneroworld.com/#nodes) has some great resources for finding nodes. One of the easiest methods would be to use a public remote node run by moneroworld, but they have a tool for finding random nodes too.
+First, you will need to find a public remote node to connect to. The website [moneroworld.com](https://moneroworld.com/#nodes) has some great resources about remote nodes, and the website [monero.fail](https://monero.fail) has an actively maintained list of functioning remote nodes. 
```

Comment:
not sure if it matters but there's a trailing space here

Reply:
definitely. this would explain my frustrations with trying to get the translations to match string / replace, thanks! will re-do things after changing